base/show.jl

```diff
@@ -1757,6 +1762,18 @@ function showarray(io::IO, X::AbstractArray, repr::Bool = true; header = true)
     end
 end
 
+"""
+    showcompact(io::IO=STDOUT, x)
```

Comment:
list as two separate signatures, since optional argument syntax only works for trailing arguments